osquery/tables/applications/posix/docker.cpp

```diff
@@ -1018,7 +1018,7 @@ QueryData genVolumeLabels(QueryContext& context) {
 /**
  * @brief Image layer extractor for docker_image_layers table
  */
-void getImageLayers(std::string image_id, QueryData& results) {
+void getImageLayers(const std::string image_id, QueryData& results) {
```

Comment:
```suggestion
void getImageLayers(const std::string& image_id, QueryData& results) {
```

Reply:
Crap, missed that from you original comment @theopolis . Sorry. Fixing